style.css

```diff
@@ -678,46 +841,74 @@ label {
     display: none;
   }
 
-  .about_description {
-    width: 68vw;
+  /* POPUP STARTS */
+
+  .popup-content {
+    padding: 0;
   }
 
-  .about_container {
-    background-image:
-      url(./assets/background/illustration-about-me-2-desktop.svg),
-      url(./assets/background/illustration\ 1\ about\ me\ desktop.svg);
-    background-size: 15%, 85%;
+  .popup-content__wrapper {
+    grid-template-columns: auto;
+    grid-template-areas:
+      "..  close"
+      "img  img"
+      "title buttons"
+      "tags  tags"
+      "description description";
+    padding: 2rem 20px;
   }
 
-  .about__list {
+  #popup__card__tags li:nth-child(n + 4) {
     display: flex;
-    flex-basis: 28.3333%;
-    flex-wrap: wrap;
-    justify-content: center;
   }
 
-  .skills {
-    grid-area: 2/1/3/3;
+  .popup__description {
+    max-width: 100%;
+  }
+
+  #popup__close__button {
+    grid-area: close;
+    justify-self: end;
+    margin: 0 0 1rem 0;
+    background: #ebecf0;
+    mix-blend-mode: multiply;
+    border-radius: 8px;
+    position: relative;
+  }
+
+  #popup__close__button .bar {
+    background-color: #67798e;
+  }
+
+  .popup__buttons__container {
+    width: 100%;
+    grid-area: tags;
+    flex-flow: row wrap;
+    justify-content: flex-end;
+    align-self: center;
+    margin-top: 2.5rem;
+  }
```

Comment:
- [ ] Similarly, for the project button, kindly make sure to place them according to the given design guidelines as suggested [here](https://github.com/mateoburvi/Portfolio/pull/12#discussion_r1249221892) to get a design close to the Figma template as much as possible 💪📌

Your design | Figma design 
---| ---|
![image](https://github.com/mateoburvi/Portfolio/assets/106119105/ddda89d1-0f08-4e7b-8343-d3c6f3c34447) | ![image](https://github.com/mateoburvi/Portfolio/assets/106119105/94fb4e68-f841-4f4f-aeac-5f6c06f7c730)